tests/models/speech_to_text/test_feature_extraction_speech_to_text.py

```diff
@@ -163,10 +163,10 @@ def test_dither(self):
 
         # compute features
         input_features_no_dither = feature_extractor_no_dither(
-            np_speech_inputs, padding=True, return_tensors="np"
+            np_speech_inputs, padding=True, return_tensors="np", sampling_rate=dict_no_dither["sampling_rate"]
```

Comment:
(the `sampling_rate` arg is redundant when it is manually set at init time, but we throw a warning nevertheless. warnings make flake-finder runs much slower :D)